README.md

```diff
@@ -1,13 +1,15 @@
 JMX Exporter
 =====
 
-JMX to Prometheus exporter.
-
-A Collector that can configurably scrape and expose mBeans of a JMX target. It
-meant to be run as a Java Agent, exposing an HTTP server and scraping the local
-JVM.
-
-This can be also run as an independent HTTP server and scrape remote JMX targets.
+JMX to Prometheus exporter: a collector that can configurably scrape and
+expose mBeans of a JMX target.
+
+This exporter is intended to be run as a Java Agent, exposing a HTTP server
+and serving metrics of the local JVM. It can be also run as an independent
+HTTP server and scrape remote JMX targets, but this has various
+disadvantages, such as the inability to expose core JVM metrics (e.g.,
```

Comment:
technically you can get all the JVM metrics, as they're just mBeans. Process metrics you can't though.

I'd mention harder to configure for the http server too.

Reply:
What do you think of this version?